extensions/extension.go

```diff
@@ -0,0 +1,79 @@
+package extensions
+
+import (
+	"math"
+
+	"grol.io/grol/object"
+)
+
+var (
+	initDone  = false
```

Comment:
Maybe an atomic.Bool?

Or sync.Once in init?

Reply:
none of the code is thread safe so far and even if it was, Init() style should be called from the main thread (like init() is)

Reply:
OK, maybe a comment about it then

Reply:
did add some good